server/src/main/java/org/spine3/server/entity/AbstractVersionableEntity.java

```diff
@@ -274,7 +274,8 @@ public LifecycleFlags getLifecycleFlags() {
      *
      * @return {@code true} if the entity is archived, {@code false} otherwise
      */
-    protected boolean isArchived() {
+    @Override
+    public boolean isArchived() {
```

Comment:
Did we consider declaring these methods (which became `public` as discussed) as `final`?

Let's discuss. My idea is to prevent the user from overriding the implementation of the "internal" fields, that we expose as a part of framework.